bridge_adaptivity/config/routing.py

```diff
@@ -0,0 +1,11 @@
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter, URLRouter
+import module.routing
```

Comment:
local packages are the third group of imports, please separate it from the second one. 

BTW I would import `websocket_urlpatterns` from the `module.routing`